covid-19-support/src/components/Header.vue

```diff
@@ -123,4 +137,14 @@ export default {
 .ncblue {
   color: $teal;
 }
+
+.icons a {
+  color: rgba(255, 255, 255, 0.5);
```

Comment:
The colors need to use the theme colors, I would think the default color already used for the fonts in the header should work?